src/providers/TokensProvider.tsx

```diff
@@ -105,21 +105,20 @@ export default function TokensProvider({
       )
       if (!tokenInfos) return
       const tokenListsRes = await fetch(
-        "https://gateway.ipfs.io/ipns/tokens.uniswap.org",
+        "https://tokens.coingecko.com/uniswap/all.json",
       )
       // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-      const tokenLists = await tokenListsRes.json()
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      const tokenListsTokensTyped = tokenLists.tokens as { address: string }[]
-      const tokenListsTokens = tokenListsTokensTyped.map((token) =>
+      const tokenLists: { tokens: { address: string }[] } =
+        await tokenListsRes.json()
+      const tokenListsTokenAddrs = tokenLists.tokens.map((token) =>
         token.address.toLowerCase(),
       )
       Object.keys(tokenInfos).forEach((address) => {
         ;(tokenInfos[address] as BasicToken).isLPToken = lpTokens.has(address)
         ;(tokenInfos[address] as BasicToken).typeAsset =
           tokenType[address] ?? PoolTypes.OTHER
         ;(tokenInfos[address] as BasicToken).isOnTokenLists =
-          tokenListsTokens.some((tokenAddr) => tokenAddr === address)
+          tokenListsTokenAddrs.some((tokenAddr) => tokenAddr === address)
```

Comment:
Could we convert `address` to lower case?

Reply:
that's happening on line 115/114